.github/workflows/build_dockers_debian11.yml

```diff
@@ -102,3 +102,72 @@ jobs:
       if: success()
       run: |
           echo '{"text":":octopus: Github Actions: build_dockers succeeded. New navitia ${{env.navitia_tag}} images available."}' | http --json POST ${{secrets.SLACK_NAVITIA_TEAM_URL}}
+
+  build_navitia_arm64:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          submodules : recursive
+          fetch-depth: 0
+
+      - name: Set up QEMU
```

Comment:
@prhod Using QEMU (emulation layer) to build ARM64 images takes a very long time. Do you think it would be feasible to deploy Github runners on ARM64 so the compilation would be straightforward?